Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/JobDataInteraction/EntityPersistentDataWriter.cs

```diff
@@ -30,5 +25,20 @@ internal EntityPersistentDataWriter(ref UnsafeParallelHashMap<Entity, TData> loo
             get => m_Lookup[entity];
             set => m_Lookup[entity] = value;
         }
+
+
+        internal EntityPersistentDataWriter(ref UnsafeParallelHashMap<Entity, TData> lookup)
+        {
+            m_Lookup = lookup;
+        }
+
+        /// <inheritdoc cref="UnsafeParallelHashMap{TKey,TValue}.Remove"/>
+        public bool Remove(Entity entity) => m_Lookup.Remove(entity);
+
+        /// <inheritdoc cref="UnsafeParallelHashMap{TKey,TValue}.TryGetValue"/>
+        public bool TryGet(Entity entity, out TData data) => m_Lookup.TryGetValue(entity, out data);
+
+        /// <inheritdoc cref="UnsafeParallelHashMap{TKey,TValue}.ContainsKey"/>
+        public bool Contains(Entity entity) => m_Lookup.ContainsKey(entity);
```

Comment:
```suggestion
        public bool ContainsKey(Entity entity) => m_Lookup.ContainsKey(entity);
```
Minor - consistency in function name